internal/state/config/options.go

```diff
@@ -142,6 +146,134 @@ var (
 		(DefaultPreferenceFlags&^OptionFlagPFlag)|OptionFlagSlice,
 		nil,
 	)
+
+	OptionSortCertificate = newOpt(
+		"sort.certificate",
+		"Default sorting for Certificate resource",
+		[]string{"id:asc"},
+		(DefaultPreferenceFlags&^OptionFlagPFlag)|OptionFlagSlice|OptionFlagHidden,
+		nil,
+	)
+
+	OptionSortDatacenter = newOpt(
+		"sort.datacenter",
+		"Default sorting for Datacenter resource",
+		[]string{"id:asc"},
+		(DefaultPreferenceFlags&^OptionFlagPFlag)|OptionFlagSlice|OptionFlagHidden,
+		nil,
+	)
+
+	OptionSortFirewall = newOpt(
+		"sort.firewall",
+		"Default sorting for Firewall resource",
+		[]string{"id:asc"},
+		(DefaultPreferenceFlags&^OptionFlagPFlag)|OptionFlagSlice|OptionFlagHidden,
+		nil,
+	)
+
+	OptionSortFloatingIP = newOpt(
+		"sort.floating-ip",
+		"Default sorting for Floating IP resource",
+		[]string{"id:asc"},
+		(DefaultPreferenceFlags&^OptionFlagPFlag)|OptionFlagSlice|OptionFlagHidden,
+		nil,
+	)
+
+	OptionSortImage = newOpt(
+		"sort.image",
+		"Default sorting for Image resource",
+		[]string{"id:asc"},
+		(DefaultPreferenceFlags&^OptionFlagPFlag)|OptionFlagSlice|OptionFlagHidden,
+		nil,
+	)
+
+	OptionSortISO = newOpt(
+		"sort.iso",
+		"Default sorting for ISO resource",
+		[]string{"id:asc"},
+		(DefaultPreferenceFlags&^OptionFlagPFlag)|OptionFlagSlice|OptionFlagHidden,
+		nil,
+	)
```

Comment:
I noticed that the API does not support sorting for:

- ISOs
- Load Balancer Types
- Networks
- Server Types

I think we should remove these options and only add them if the API also supports this.